docs/README_en.md

```
@@ -311,3 +325,6 @@ const getVoiceLanguageCode = (selectLanguage: string) => {
 #### Adding README
 
 - Add a new language README (`README_fr.md`), logo usage terms (`logo_licence_fr.md`), and VRM model usage terms (`vrm_licence_fr.md`) to the `docs` directory.
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)